spectral/ruleset.yaml

```diff
@@ -9,6 +9,21 @@ functions:
 rules:
   openapi-tags: off
   operation-tags: off
+  # openapi spec rules, that return errors
+  oas3-valid-schema-example: off
+  no-$ref-siblings: off
```

Comment:
I think that one would not be required - we need to update to openapi 3.1 where this is supported. Linter should probably stop reporting it.